src/OpenTelemetry.Exporter.Geneva/Common.GenevaExporter.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <OTelSdkVersion>$(OpenTelemetryCoreLatestVersion)</OTelSdkVersion>
+    <OTelSdkVersion>$(OpenTelemetryCoreLatestPrereleaseVersion)</OTelSdkVersion>
```

Comment:
We might not even need this indirection and `EXPOSE_EXPERIMENTAL_FEATURES` constant anymore. We could consider removing it later.